packages/Metric/src/Score.tsx

```diff
@@ -36,6 +38,35 @@ const Score: React.FunctionComponent<ScoreProps> = ({
 }: ScoreProps) => {
     const stringFormatter = useLocalizedStringFormatter(intlMessages);
     const { locale } = useLocale();
+
+    const getBrand = (): string => {
+        return document.documentElement.getAttribute("data-brand") ?? "igloo";
+    };
+
+    const isWorkleap = getBrand() === "workleap";
+
+    const sizeMap = {
```

Comment:
this object can be outside the component, so it doesn't get created everytime

Reply:
you should also add " as const" at the end, so you won't have to cast it with IconProps["size"] later